example/crud.postman_collection.json

```diff
@@ -2657,6 +2657,46 @@
 				}
 			},
 			"response": []
+		},
+		{
+			"name": "add project with invalid schema",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "6f21622a-a837-44c9-8823-a82279030fd2",
+						"exec": [
+							"pm.test(\"Status code is 400\", function () {\r",
+							"    pm.response.to.have.status(400);\r",
+							"});"
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "POST",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": "{\r\n  \"projects\": [\r\n    {\r\n      \"orgId\": \"string\",\r\n      \"name\": \"string\",\r\n      \"id\": \"string\",\r\n      \"schema\":\"invalid\"\r\n    }\r\n  ]\r\n}",
```

Comment:
since this is testing for schema, give it a valid orgId that's been created

Reply:
changed it to a valid orgId